aws/components/ecs/setup.ftl

```diff
@@ -1622,6 +1622,10 @@
                 dependencies=dependencies
                 fixedName=solution.FixedName
             /]
+            
+            [#if containers?size < 1 ]
+                [@fatal message="No container available" context=resources["task"] /]
```

Comment:
Is that context useful enough to be able to identify the task or service where this has happened? 
If you looked at what is provided could you fix the issue?

Reply:
That sounds like I'm missing something obvious. I get an error message of
```
{
  "Timestamp": "2021-08-02T08:31:24.602+10:00",
  "Severity": "fatal",
  "Message": "No container available",
  "Context": {
    "Id": "ecsTaskXappXserviceecsbaseXtaskbase",
    "Name": "application-serviceecsbase-taskbase",
    "Type": "ecsTask",
    "Deployed": false
  }
}
```
With a solution file of
```
  "Tiers": {
    "app": {
      "Components": {
        "serviceecsbase": {
          "ecs": {
<snip>
            "Tasks": {
              "taskbase": {
                "Instances": {
                  "default": {
                    "deployment:Unit": "aws-task-base"
                  }
                },
<snip>
```

Reply:
From the exception message could I figure out what I need to do to fix the problem that has caused a fatal error to occur? 

I think it needs a suggestion on what to do or a hint on whats going on. Its also containers not container. You can have multiple containers in a task or service definition

Reply:
Like
```
{
  "Timestamp": "2021-08-02T16:49:20.496+10:00",
  "Severity": "fatal",
  "Message": "No container available. Add one or more containers to the following service/task",
  "Context": {
    "Id": "ecsTaskXappXserviceecsbaseXservicebase",
    "Name": "application-serviceecsbase-servicebase",
    "Type": "ecsTask",
    "Deployed": false
  }
}
```
Or am I still missing the point

Reply:
Yeah I think that covers it.